src/app/components/utility/selectphase/selectphase.component.html

```diff
@@ -1,21 +1,33 @@
-<div class="select-phase">
-  <div class="select-title heading">Select a phase:</div>
-  <div [class.inline]="publicRouter.url.indexOf('submissions') >= 0" class="phase-div" *ngFor="let item of phases">
-    <div class="card card-light" (click)="selectPhase(item)" [class.selected]="item['is_selected']">
+<span *ngIf="phaseSelectionType=='radioButton'">
+<li *ngFor="let item of phases">
```

Comment:
`span` above is used only to bind `ngIf`. We could use `ng-container` instead, as `span` in vain is getting appended to DOM. (Reference, https://stackoverflow.com/questions/38585720/how-to-apply-multiple-template-bindings-on-one-element-in-angular/43846773#43846773)

Reply:
cool